allensdk/internal/brain_observatory/time_sync.py

```diff
@@ -15,7 +15,7 @@
 REG_PHOTODIODE_MAX = 2.1 # seconds
 PHOTODIODE_ANOMALY_THRESHOLD = 0.5 # seconds
 LONG_STIM_THRESHOLD = 0.2 # seconds
-ASSUMED_DELAY = 0.0351 # seconds
+ASSUMED_DELAY = 0.0215 # seconds
```

Comment:
I'm cautiously confident that this value is better than 0.0351 based on some analysis Wayne and I did. However, there should be a mechanism in place to re-evaluate this every so often as we collect more data to make sure this is still a reasonable value.